src/app/domain/workInterval.enum.ts

```diff
@@ -0,0 +1,8 @@
+export const WorkInterval = {
+  BusyTimes: 'Busy Times',
+  EasyJets: 'Easy jets',
+  ReturningPips: 'Returning pips',
+  Sleepies: 'Sleepies',
+} as const;
+
+export type WorkIntervalEnum = (typeof WorkInterval)[keyof typeof WorkInterval];
```

Comment:
I'm just curious, why do we need an enum here? Won't a type literal do the same job, but simpler?

```
export type WorkInterval = 'Busy Times' | 'Easy jets' …